smartling-strings-api/src/main/java/com/smartling/api/strings/v2/pto/StringsCreationPTO.java

```diff
@@ -0,0 +1,53 @@
+package com.smartling.api.strings.v2.pto;
+
+import java.util.List;
+
+public class StringsCreationPTO
```

Comment:
how do I do create+authorize?
you need to add into this class
`private StringsAuthorizationPTO authorization;`

Reply:
from public doc. 
Note: You cannot authorize a string via API. Once you have created the strings, they will need to be authorized by a content owner in the Smartling Dashboard. 
so did this by intend